Apps/Examples/Examples/SiwftUI Examples/FeaturesQueryExample.swift

```diff
@@ -0,0 +1,112 @@
+import SwiftUI
+@_spi(Experimental) import MapboxMapsSwiftUI
+
+@available(iOS 14.0, *)
+struct FeaturesQueryExample: View {
+    @State private var camera = CameraState(center: .newYork, zoom: 10)
+
+    @State private var queryResult: QueryResult? = nil
+
+    var body: some View {
+            Map(camera: $camera)
+                .styleURI(.streets)
+                .onMapTapGesture(action: { _, res in
+                    queryResult = try? QueryResult(features: res.get())
+                })
+        .edgesIgnoringSafeArea(.all)
+        .sheet(item: $queryResult, onDismiss: { queryResult = nil }) { res in
+            ResultView(result: res)
+                .defaultDetents()
+        }
+    }
+}
+
+
+@available(iOS 14.0, *)
+struct QueryResult: Identifiable {
+    struct Feature: Identifiable {
+        var id = UUID()
+        var sourceId: String
+        var sourceLayer: String?
+        var name: String?
+        var category: String?
+
+        init(feature: QueriedFeature) {
+            sourceId = feature.source
+            sourceLayer = feature.sourceLayer
+            let properties = feature.feature.properties
+            name = properties?["name_en"]?.flatMap(\.asString) ?? properties?["name"]?.flatMap(\.asString)
+            category = properties?["category_en"]?.flatMap(\.asString) ?? properties?["category"]?.flatMap(\.asString)
+
+        }
+    }
+    var id = UUID()
+    var features: [Feature]
+
+    init(features: [QueriedFeature]) {
+        self.features = features.map {
+            Feature(feature: $0)
+        }
+    }
+}
+
+@available(iOS 14.0, *)
+struct ResultView: View {
+    let result: QueryResult
+    var body: some View {
+        if result.features.isEmpty {
+            Text("Nothing found 😔")
+        } else {
+            List {
+                Section {
+                    ForEach(result.features) { f in
+                        VStack(alignment: .leading) {
```

Comment:
Probably no need in `VStack` here, it doesn't seem to do anything, even without it FeatureView is correctly aligned. (at least on my device😁)
If you still need to align you can use `.frame(alignment: .leading)` modifier.